pkg/common/common.go

```diff
@@ -103,10 +104,11 @@ const (
 
 	// DefaultResyncPeriod sets a 10 minute resync period, used in the controller pkg and the controller cmd executable
 	DefaultResyncPeriod = 10 * time.Minute
-
 	// InsecureRegistryConfigMap is the name of the ConfigMap for insecure registries
 	InsecureRegistryConfigMap = "cdi-insecure-registries"
 
 	// ScratchSpaceNeededExitCode is the exit code that indicates the importer pod requires scratch space to function properly.
 	ScratchSpaceNeededExitCode = 42
+	// VolumeMode provides a constant to capture our env variable "VOLUME_MODE"
+	VolumeMode = "VOLUME_MODE"
```

Comment:
Since we are no longer passing the environment variable, we don't need this constant.